hal/halx86/acpi/busemul.c

```diff
@@ -93,10 +93,18 @@ HalpGetSystemInterruptVector_Acpi(IN ULONG BusNumber,
                                  OUT PKIRQL Irql,
                                  OUT PKAFFINITY Affinity)
 {
+#ifdef _M_AMD64
```

Comment:
don't we have HAL_APIC define or something ?

Reply:
No, it's all compiled as libraries, that are then linked together appropriately for each hal. I think this function should go to the PIC library and both should have the same name.